omnibus/config/software/fieri.rb

```diff
@@ -1,22 +0,0 @@
-#
-# Copyright 2016 Chef Software, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-name "fieri"
-license :project_license
-
-source path: File.expand_path('../../../../src/fieri', project.filepath)
-
-dependency "file"
```

Comment:
Because fieri didn't exist within supermarket's Rails root directory structure, these were the shenanigans necessary to get the fieri source located in a spot that the supermarket Gemfile expected (`../fieri`) for the omnibus build.